07-make_evoked.py

```diff
@@ -35,8 +35,6 @@ def run_evoked(subject):
         evokeds.append(epochs[condition].average())
 
     mne.evoked.write_evokeds(fname_out, evokeds)
-    
-    # look on MNE website how to plot the evoked. evoked.plot or similar 
```

Comment:
general remark: include the link to the tutorial, might be helpful for yourself later and others if they use your scripts